### DISCUSSION TITLE
'Exception in thread "Render Thread"' error

### DISCUSSION THREAD

**Discussion by DakotaPride** [FabricMC/fabric-api#1770] [Mod Dev Support]
Just as the title says, I'm not sure what I do to fix this. I am new to this, so sorry if this is simple to fix. If you need more information, I will gladly provide it as much as I can. It doesn't list where or why this error occurred, so the information is limited.

**Comment by Juuxel** [comment]
Could you post the full game log? This error message alone doesn't really give any hints.

**Reply by DakotaPride** [reply]
![Screenshot (385)](https://user-images.githubusercontent.com/82760983/136852211-c52980d5-a61a-4d43-8367-2c5383c1b3f2.png)
Sorry if this doesn't help
